packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -340,7 +338,7 @@ function updateDOMProperties(
     } else if (propKey === DANGEROUSLY_SET_INNER_HTML) {
       setInnerHTML(domElement, propValue);
     } else if (propKey === CHILDREN) {
-      setTextContent(domElement, propValue);
+      domElement.textContent = propValue;
```

Comment:
Likewise, should we stringify?

Reply:
I think the string if you happens in diffProperties for update, so the value is already a string here

Reply:
Got it. In any case, this looks consistent with the old behavior. 👍